src/assisted_test_infra/download_logs/download_logs.py

```diff
@@ -144,6 +144,10 @@ def download_logs(
                     cluster["id"], cluster_file, os.path.join(output_folder, "cluster_files", cluster_file)
                 )
 
+        manifests_path = os.path.join(output_folder, "cluster_files", "manifests")
+        recreate_folder(manifests_path)
+        client.download_manifests(cluster["id"], manifests_path)
```

Comment:
This function is long enough, move manifest l logic to different function.
e.g.
```suggestion
        self.download_manifests(output_folder, cluster["id"])
        ....
def download_manifests(output_folder: str, cluster_id: str):
        manifests_path = os.path.join(output_folder, "cluster_files", "manifests")
        recreate_folder(manifests_path)
        client.download_manifests(cluster_id, manifests_path)
```